core/src/main/java/io/github/tcdl/messages/Topics.java

```diff
@@ -13,32 +13,12 @@
     private final String response;
 
     @JsonCreator
-    private Topics(@JsonProperty("to") String to, @JsonProperty("response") String response) {
+    public Topics(@JsonProperty("to") String to, @JsonProperty("response") String response) {
```

Comment:
I would define another constructor that accepts only "to" topic because "response" is optional and very often is null